packages/dialtone-vue2/components/popover/popover.vue

```diff
@@ -946,16 +946,61 @@ export default {
       }
     },
 
+    getReferenceClientRect (appendTo, error) {
+      const anchorReferenceRect = this.anchorEl.getBoundingClientRect();
+
+      if (this.appendTo !== 'root' || error) return anchorReferenceRect;
+
+      const iframe = appendTo.getElementsByTagName('iframe');
+      const iframeReferenceRect = iframe[0].getBoundingClientRect();
+
+      if (!iframe || this.anchorEl.ownerDocument === iframe[0].ownerDocument) return anchorReferenceRect;
```

Comment:
Lets safe access here on almost everything to avoid the reference error.

`this.anchorEl?.getBoundingClientRect();`
`iframe[0]?.getBoundingClientRect();`
`this.anchorEl?.ownerDocument`
`iframe[0]?.ownerDocument`